pom.xml

```diff
@@ -449,8 +449,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.8.1</version>
                     <configuration>
-                        <source>1.8</source>
-                        <target>1.8</target>
```

Comment:
Is this change supposed to be in this pull request?

Reply:
@dkayiwa no. I removed that change